cmd/srcd/cmd/root.go

```diff
@@ -88,37 +88,89 @@ func initConfig() {
 
 var logMsgRegex = regexp.MustCompile(`.*msg="(.+)"`)
 
-func logAfterTimeout(header string) chan struct{} {
-	logs, err := daemon.GetLogs()
-	if err != nil {
-		logrus.Errorf("could not get logs from server container: %v", err)
+func logAfterTimeout(msg string, timeout time.Duration) func() {
```

Comment:
this function isn't used anywhere.

Reply:
I left it just in case it could be useful. I just treated this package as a sort-of utils package providing different types of logging with timeout.

Reply:
ok